plugin/trino-iceberg/src/test/java/io/trino/plugin/iceberg/TestIcebergPlugin.java

```diff
@@ -74,6 +74,7 @@ public void testGlueMetastore()
                 "test",
                 Map.of(
                         "iceberg.catalog.type", "glue",
+                        "hive.metastore.glue.catalogid", "123",
```

Comment:
add as a new test case (within this test method), so that here we continue to test with minimal required configs